packages/ui/src/app/App.stories.tsx

```diff
@@ -408,3 +413,44 @@ export const BuyMembershipTxFailure: Story = {
     expect(await modal.findByText('Some error message'))
   },
 }
+
+// ----------------------------------------------------------------------------
+// Test Emil Subsciption Modal
+// ----------------------------------------------------------------------------
+export const EmailSubscriptionModalDecline: Story = {
+  args: {
+    isLoggedIn: true,
+    hasMemberships: true,
+    hasAccounts: true,
+    hasFunds: true,
+    hasWallet: true,
+    isRPCNodeConnected: true,
+    hasRegisteredEmail: false,
+  },
+  play: async ({ canvasElement }) => {
+    const screen = within(canvasElement)
```

Comment:
For a slight performance improvement but also improving the test accuracy, whenever looking for an element in a modal restrict the search to the modal container:
```suggestion
    const modal = withinModal(canvasElement)
```